src/linkml/include_participant.yaml

```diff
@@ -147,16 +156,21 @@ slots:
     title: Family Type
     range: enum_family_type
     required: true
-  father_id:
-    definition_uri: include:father_id
-    description: Participant ID for Participant's father
-    title: Father Id
-    range: string
+  relationship_type:
+    definition_uri: include:relationship_type
+    description: Type of familial relationship
+    title: Relationship Type
+    range: enum_relationship_type
   has_participant:
     definition_uri: include:has_participant
     description: Link to a Participant
     title: Has Participant
     range: Participant
+  has_relationship:
+    definition_uri: include:has_relationship
+    description: Link to a Relationship
```

Comment:
I might add a bit more detail here, e.g.  "A link to a FamilyRelationship object that describes the nature of the Participant's relationship to another family member."